client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -18,6 +18,8 @@ angular
         }
 
         canDeleteSSL () {
-            return this.$scope.hosting != null && this.$scope.hosting.hasHostedSsl && this.alreadyHasSSL();
+            // hasHostedSsl is true only for legacy SSL offers
+            // the new system is through the ssl API
+            return this.$scope.hosting != null && (this.$scope.hosting.hasHostedSsl || this.alreadyHasSSL());
```

Comment:
What do you think about : 

`canDeleteSSL () => _.get(this, "$scope.hosting.hasHostedSsl") || this.alreadyHasSSL();`

Reply:
:+1: